Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -1263,6 +1269,72 @@ private static void OnIsValidPropertyChanged(DependencyObject d, DependencyPrope
             }
         }
 
+        /// <summary>
+        /// Gets or sets the threshold range that governs when the DataGrid class will begin to prefetch more items.
+        /// </summary>
+        /// <returns>
+        /// The loading threshold, in terms of pages.
+        /// </returns>
+        public double IncrementalLoadingThreshold
```

Comment:
Alphabetical order of properties should be maintained.

Reply:
Sorry, wasn't aware that this applied also to properties and methods. Will take note of it for the future.